src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
-        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
+        maybeExecutionStore = Option(new ExecutionStore(binding))
         maybeSymbolStore = Option(new SymbolStore(binding, actualInputs))
         maybeBackend = Option(backend)
-        executionStatusStore.runnableCalls.foreach { call =>
-          startCallActor(symbolStore, call)
-        }
-        primeMover ! Started
+        // Poll for runnable calls.
+        system.scheduler.schedule(
+          initialDelay = 0 milliseconds,
+          interval = 5 seconds,
+          receiver = self,
+          message = CheckExecutionStatus)(system.dispatcher)
+
         context.become(started)
+        primeMover ! Started
+
 
-      case bad @ _ =>
+      case bad@_ =>
         sender ! InvalidOperation(s"Received message $bad before any Start message")
     }
   }
 
+  private def createRequiredInputs(call: Call): Unit = {
```

Comment:
Per discussion with Scott, this is very cool but probably more than I can bite off for a cleanup pass.  We should try to steer Anthony in developing user stories that would motivate reworking the way inputs are stored along the lines of Scott's comments above.